extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ClassConfigPropertiesUtil.java

```diff
@@ -286,16 +292,6 @@ private static ResultHandle populateConfigObject(ClassInfo configClassInfo,
                                     MethodDescriptor.ofMethod(configObjectClassStr, getterName, fieldTypeStr),
                                     configObject);
                         }
-
-                        /*
-                         * If the getter value is null (meaning that no default was set)
-                         * throw the exception that MP Config would.
-                         * If the value is not null, then we don't need to do anything since the class will already
-                         * contain the default value the user intended to use
-                         */
-                        isPresentFalse.ifNull(defaultValue).trueBranch()
```

Comment:
Can you please explain why this needs to be removed?

Reply:
Since missing properties are detected at build time with this PR, can this runtime exception still be thrown? It looks like the answer is no to me, but maybe I'm missing something.

Reply:
Hold on, the `build time` part is scaring me :)

You mean application startup time, right?

Reply:
Yes sorry, I meant startup time :)

Reply:
OK, I thought that I had massively missunderstood this PR :)

Reply:
My fingers answered while my brain was focused on another matter, my bad :D

Reply:
NP! Just wanted to get things straight.